lib/test.js

```diff
@@ -92,22 +102,50 @@ function run(state, callback) {
     process.env.DISPLAY = program.display ? program.display : ':0';
   }*/
 
+  if(command.sauceConfig) {
+    remoteSelenium = true;
+    if(!(process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY)) {
+      return callback(new Error(
+        'Missing environment variables: SAUCE_USERNAME, SAUCE_ACCESS_KEY'));
+    }
+    bedrock.config.protractor.meta = {
+      build: command.sauceBuild || null,
+      name: command.sauceName || null,
+      tags: []
+    };
+    if(command.sauceTag) {
+      command.sauceTag.split(',').forEach(function(t) {
+        bedrock.config.protractor.meta.tags.push(t);
+      });
```

Comment:
Not sure if this is better given the really long target array, but this could be "shortened" to:

```js
bedrock.config.protractor.meta.tags.apply(
  bedrock.config.protractor.meta.tags, command.sauceTag.split(','));
```